services/workers/src/test/java/org/sagebionetworks/table/worker/MaterializedViewUpdateWorkerIntegrationTest.java

```diff
@@ -343,6 +348,94 @@ public void tesMaterializedViewIsRebuildWhenDefiningSQLIsChanged() throws Except
 		});
 	}
 
+	@Test
+	public void testDeletesEntityAndMoveToTrashDeletesTableSatus() throws Exception {
```

Comment:
we can use just one test for MV that a has a dependency table that gets moved to the trash can.   All other integration tests for each type can be removed.